### PR TITLE
feat(fonts): add Brandon Grotesque and Ugo Color

### LINKED ISSUE #12: feat(all): import all font assets

### PR DESCRIPTION
**Pull request by selynna**
closes #12

### CODE REVIEW

File: src/layouts/index.css [hunk 1]
```diff
@@ -1 +1,130 @@
-/* Fonts */
+@font-face {
+    font-family: 'UGO COLOR';
+    src: url('UGO-COLOR.eot');
```
Comment (mianuddin) [inline src/layouts/index.css]: The fonts are not in the `src/layouts/` folder, you have to specify their path, which in this case is `src/assets/fonts/`. This applies to all of the other `url`s here. Travis CI only lints, which does not catch errors like these. However, you will be presented with an error if you try to build.

screen shot 2018-09-16 at 22 37 23

Feel free to reference this:
https://github.com/slohacks/club/blob/c6ac5479c5c4b2e0f0030c929e9a07534d2c59f9/src/layouts/index.css#L4

**Review by mianuddin — CHANGES_REQUESTED** [review]
The paths are incorrect, reference my comment for more details.

Other friendly reminders:
* your pull request title will become the squashed commit message, so you should update it to conform to the standard
* typically, changes should not be done on your master branch
    - this may force you to rebase once the PR is merged in

File: src/layouts/index.css (hunk 1 above)
Comment (selynna) [reply src/layouts/index.css]: I was having some issues with other branches and had to cherry-pick and rebase a couple of times. Going to keep this on master for the time being and rebase after merged (and work on a different branch for future pr's)

Comment (mianuddin) [reply src/layouts/index.css]: The branch issue won't affect the `slohacks` repo, as it is on your fork. Just a heads up for your own sanity lol.

Comment (selynna) [reply src/layouts/index.css]: Updated in https://github.com/slohacks/event-dev/pull/18/commits/c90ef97a9cf63c9b34cd3f6937b8dba81b29d9f3 - resolving conversation

Comment (selynna) [reply src/layouts/index.css]: @mianuddin also yeah, i know it won't affect the upstream repo, but i was having trouble with branches on local fork and needed to cherry pick master/rebase branches. ended up getting super weird so I decided to stick with merging this on master and fixing it after

Comment (selynna) [reply src/layouts/index.css]: but yes, thanks for the heads up - do appreciate it :-)

**Review by mianuddin — APPROVED** [review]
Looks good.

**Pull request merged**